src/components/BTextarea/BTextarea.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div :class="classes">
+    <label v-if="!hideLabel" :for="id">
+      {{ label }}<span v-if="required" aria-label="Required">*</span>
+    </label>
+    <textarea
+      v-model="inputValue"
+      v-bind="inputAttributes"
+    />
+    <div class="b-input-group--error">
+      <span></span>
+    </div>
+  </div> 
+</template>
+
+<script>
+export default {
+  name: 'b-textarea',
+  props: {
+    id: {
+      type: String,
+      required: true
+    },
+    name: {
+      type: String,
+      required: true
+    },
+    cols: {
+      type: Number,
+      default: 30
+    },
+    rows: {
+      type: Number,
+      default: 5
+    },
+    placeholder: String,
+    hideLabel: Boolean,
+    label: {
+      type: String,
+      required: true
+    },
+    required: {
+      type: Boolean,
+      default: false
+    },
+    disabled: {
+      type: Boolean,
+      default: false
+    }
+  },
+  computed: {
+    classes () {
+      return {
+        'b-input-group': true,
+        'b-input-group--disabled': this.disabled
+      }
+    },
+    inputValue: {
+      get () {},
+      set (value) {
+        this.$emit('input', value)
+      }
+    },
+    inputAttributes () {
+      return {
+        id: this.id,
+        name: this.name,
+        cols: this.cols,
+        rows: this.rows,
+        placeholder: this.placeholder,
+        required: this.required,
+        disabled: this.disabled,
+        hideLabel: this.hideLabel
```

Comment:
Seems like this should be `aria-label: this.hideLabel ? "" : this.label`

Reply:
I'm happy to be outvoted on this one too, but it's my preference that we just have these up in the template rather than abstracted down into this function. It's lines here or lines there and I'd rather have them up there, but that's a personal preference.

Reply:
The actual issue I'm addressing is for accessibility if "hideLabel" is true, the `aria-label` attribute needs to be set on the textarea. Or actually the `title` attribute would be better because that will also show up on hover.

@mlear @fobesjason @ghost- What are your votes regarding attributes being bound directly in the template or in a computed function?

Reply:
i think I would vote directly in the template as it makes the most sense to me, but I can go either way.

Reply:
I don't really like having every prop named in the template. It just looks super busy, especially if there are a lot of props.

Reply:
Also, accidentally took that out.

Reply:
@eleanormal I would prefer it in the template.  It gives more lexical context than being in a 
function.